tutorials/nameservice/base/x/auction/codec.go

```diff
@@ -0,0 +1,23 @@
+package auction
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/codec/legacy"
+	types "github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/msgservice"
+)
+
+// RegisterLegacyAminoCodec registers the necessary interfaces and concrete types
+// on the provided LegacyAmino codec. These types are used for Amino JSON serialization.
+func RegisterLegacyAminoCodec(cdc *codec.LegacyAmino) {
+	legacy.RegisterAminoMsg(cdc, &MsgBid{}, "namespace/MsgReserve")
+}
```

Comment:
dont think this is needed anymore

Reply:
removing this results in an error from `NewAppModule`complaining its necessary: `missing method RegisterLegacyAminoCodec`